dist/preview release/recast.js

```diff
@@ -0,0 +1,38 @@
+

```

Comment:
Where is the wasm file? Is it embedded or something?

Reply:
I compiled recast to js, not wasm.

Reply:
asm.js then? Is there a plan to use wasm?

Reply:
The recastjs build system supports outputting JS or WASM. I don't know which one is preferable.

Reply:
wasm as it is faster

Reply:
I get an issue with WASM output with an undefined symbol at runtime. It seems pretty hairy.  I prefer to keep this PR using JS and update with wasm in new PR.